spec/exchanges/c2cx/integration/market_spec.rb

```diff
@@ -0,0 +1,110 @@
+    expect(hashed_pairs).to include({base: 'USDT', target: 'DRG', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'ETC', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'ETH', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'FUN', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'LTC', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'SKY', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'TNB', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'UCASH', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'ZEC', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'ZRX', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'OMG', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'OMG', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'RCN', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'RCN', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'XRP', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'XRP', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'EOS', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'EOS', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'WAX', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'WAX', market: market})
+    expect(hashed_pairs).to include({base: 'DRG', target: 'XLM', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'XLM', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'ETC', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'LTC', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'BTG', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'ZEC', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'TRIA', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'BEZ', market: market})
+    expect(hashed_pairs).to include({base: 'BTC', target: 'SLRM', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'OMG', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'RCN', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'XRP', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'EOS', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'WAX', market: market})
+    expect(hashed_pairs).to include({base: 'USDT', target: 'XLM', market: market})
```

Comment:
good idea trying to test every pair however it is not necessary as exchanges add/remove pairs all the time.